yb-voyager/src/constants/constants.go

```diff
@@ -51,3 +51,10 @@ const (
 const (
 	OBFUSCATE_STRING = "XXXXX"
 )
+
+// constants for --on-primary-key-conflict flag values
+const (
+	PRIMARY_KEY_CONFLICT_ACTION_ERROR  = "ERROR"
+	PRIMARY_KEY_CONFLICT_ACTION_IGNORE = "IGNORE"
+	PRIMARY_KEY_CONFLICT_ACTION_UPDATE = "UPDATE"
```

Comment:
let's not support UPDATE for now. We will have to deprecate `--enable-upsert` and then also apply the session variables for upsert mode.

Reply:
followup PR